app/app.iml

```diff
@@ -4,6 +4,8 @@
     <facet type="android-gradle" name="Android-Gradle">
```

Comment:
revert app/app.iml

Reply:
Should we add app/app.iml to the .gitignore file?

Reply:
sounds good to me!